### PR TITLE
fix: hide sidebar when `custom` tool active

### PR DESCRIPTION
**Pull request by dwelle**

### CODE REVIEW

File: src/element/showSelectedShapeActions.ts [hunk 1]
```diff
@@ -7,9 +7,10 @@ export const showSelectedShapeActions = (
...
   Boolean(
-    !appState.viewModeEnabled &&
+    (!appState.viewModeEnabled &&
+      appState.activeTool.type !== "custom" &&
       (appState.editingElement ||
-        getSelectedElements(elements, appState).length ||
         (appState.activeTool.type !== "selection" &&
-          appState.activeTool.type !== "eraser")),
+          appState.activeTool.type !== "eraser"))) ||
+      getSelectedElements(elements, appState).length,
```
Comment (dwelle) [inline src/element/showSelectedShapeActions.ts:15]: this shuffle is unrelated, just for perf

**Review by ad1992 — APPROVED** [review]
